src/main/java/com/terraforged/mod/server/command/TerraCommand.java

```diff
@@ -80,9 +82,17 @@
 @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE)
 public class TerraCommand {
 
+    private static HashMap<UUID, Integer> searchId;
```

Comment:
The map needs cleaning up after a player disconnects. Idk if command execution & the disconnect event occur on the same thread so that needs to be checked and the map made thread safe accordingly (Collections.synchronizedMap would probably be fine).